docs/user/android-build-configuration-options.md

```diff
@@ -23,3 +23,20 @@ The `metrics.yaml` can be added to the dependency itself by calling this on each
 ```groovy
 variant.packageLibraryProvider.get().from("${topsrcdir}/path/metrics.yaml")
 ```
+
+## `gleanGenerateMarkdownDocs`
+
+The Glaen SDK can automatically generate Markdown documentation for metrics and pings defined in the registry files.
```

Comment:
```suggestion
The Glean SDK can automatically generate Markdown documentation for metrics and pings defined in the registry files.
```

Nice typo

Reply:
Does this mean the Kotlin code is not generated? Or is both generated? I'd like this to be made more clear here.